data/os/Ubuntu.yaml

```diff
@@ -0,0 +1,10 @@
+---
+ca_cert::supported:             true
+ca_cert::trusted_cert_dir:      '/usr/local/share/ca-certificates'
+ca_cert::update_cmd:            'update-ca-certificates'
+ca_cert::cert_dir_group:        'staff'
```

Comment:
Why `staff` as default? The ca-certificates package creates this directory with group root ...
```
root@cbe1f3cd0871:/# ls -l /usr/local/share/
total 0
drwxr-xr-x. 2 root root 6 May  9 17:31 ca-certificates
```